master/static/migrations/20220829162026_set-raw-metrics-archive.tx.up.sql

```diff
@@ -0,0 +1,10 @@
+UPDATE raw_steps update_steps
+SET archived = TRUE
+FROM (SELECT DISTINCT trial_id, total_batches, trial_run_id FROM raw_steps outer_steps
+    WHERE archived IS FALSE AND 
+        trial_run_id < (SELECT MAX(inner_steps.trial_run_id) FROM raw_steps inner_steps WHERE 
```

Comment:
in the bug, wasn't the "right" one the one with a lower run id?

also this feels like it would archive more than intended (would all distinct steps less with a run id less than the highest get archived?)

Reply:
Yep you are correct on both points.

The old query def would have updated more though I don't really understand what is wrong but no point in reasoning about incorrect code. Interesting on latest master there is only one record with this bug ```4099``` distinct UUIDs vs ```4100``` records. 

For the new query tried to use ```checkpoints_view``` to minimize the amount of ```raw_steps``` we have to touch